### ISSUE TITLE
Update decidim-design repo with current real design

### ISSUE THREAD

**Issue by Xfolchf** [decidim/design#158]

**Comment by furilo** [comment]
In some hallway conversations with @xabier @andreslucena we drafted the following ideas: 

- Put in an archive state (and reflect that in the repo) the screens, which are outdated, and doesn't make sense to update in the repo because it's an expensive effort
- Keep updating the Pattern Library section
- If we design a whole new screen, we'll add it there

Do we all agree with this plan?

Btw, what is the deploy process for this repo? It's automated from commits to the repo or is there any additional steps involved?

cc @decidim/product

**Comment by xabier** [comment]
@furilo I totally agree! go ahead! I will update the main issue of this thread with your specs and details.
